status/ss13status.py

```diff
@@ -41,6 +41,7 @@ def __init__(self, bot):
             "new_round_channel": None,
             "admin_notice_channel": None,
             "mentor_notice_channel": None,
+            "ban_notice_channel": None,
```

Comment:
I'm not sure if we need another channel for these, but it's probably OK for now.